### ISSUE TITLE
Icons no longer change color based on state ... but only in entity card??

### ISSUE THREAD

**Issue by i00** [home-assistant/home-assistant-polymer#4821]
## The problem

In 0.105 the functionality of entity highlighting was changed as mentioned [here](https://www.home-assistant.io/blog/2020/02/05/release-105/), to only change the color of light icons when their state was on (which is good ... because I like my cat NOT being yellow when she is at home :P ) ... however this only applies to the entities on config screens and with the entity card ... on the button card they are still colored.  Maybe the background of the button should be slightly darkened (like color:# 00000044 (black with 75% opacity; and had to put a space after the hash because of markdown)), to keep inline with documentation.

## Environment

- Home Assistant release with the issue: 0.105
- Last working Home Assistant release (if known): 0.104.3 ...
  **technically** still happened in versions < 0.105 ... but this behaviour was standard in those versions and should have been changed for > 0.105
- Operating environment (Hass.io/Docker/Windows/etc.): 
  Docker
- Integration causing this issue: Entity Button Card
- Link to integration documentation on our website: https://www.home-assistant.io/lovelace/entity-button/

## Problem-relevant `configuration.yaml`

```yaml
N/A
```

## Traceback/Error logs

```txt
N/A
```

## Additional information
N/A

**Comment by SeanPM5** [comment]
This was an intentional change.

The button card does not display state as text, so the icon coloring is necessary to determine state.

**Comment by Mariusthvdb** [comment]
well, it makes the use of device_class much less obvious. https://www.home-assistant.io/integrations/binary_sensor/

as explained there it should also change to color of the icon, which is one of the main reasons for using device_class..

hope this gets re-evaluated very soon. even though this issue was closed with great haste.

**Comment by bramkragten** [comment]
There is an option to enable it, so if you want it back you can
https://www.home-assistant.io/lovelace/entities/#state_color

**Comment by Mariusthvdb** [comment]
that is hardly the same isn't it? that is per entity card, and not globally. This new behavior is causing people that use device_class, and expect the icons to change color accordingly, lots of extra work (and lines in the config...) You could reply that the device_sets the icon, and nu so much the state color. Indeed, but state_color was the immediate visual feedback for, well, state ;-) Which has now gone, turning the native default HA frontend into a bleak and not very informative one.

I am purposely not saying that I want it to be as it has always been, no progress without change, I know. 
But, in this case, the coloring did exactly what we wanted it to do, and the change is rather unfortunate, and counterintuitive to what many users in the community expect from their home-automation system: have immediate visual feedback between on/off.

Long story short:
Wouldn't it be possible to have a global selector allow for de-activating icon state coloring? That way you could have it both ways, and most of all, have the user decide which behavior is desirable.

**Comment by bramkragten** [comment]
No, that is not possible. You can do it card based or entity based.

**Comment by i00** [comment]
I personally think that the colouring should be consistent, regardless of what happens including on entity buttons.. 

I think that if the state_color is disabled, this should affect both the entity buttons and entity rows for consistency, as mentioned in my original post I think there should be some other way to determine if it is on or off.

**Comment by bramkragten** [comment]
So you are asking for the option `state_color` for other cards like `entity-button`?
